bundles/GlossaryBundle/src/Migrations/Version20221215071650.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pimcore\Bundle\GlossaryBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Pimcore\Db;
+use Pimcore\Model\Tool\SettingsStore;
+
+/**
+ * Checking if glossary tables already exist
+ */
+final class Version20221215071650 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'In case the glossary table already exists, it marks the GlossaryBundle as installed';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $tableExists = Db::get()->fetchOne('SHOW TABLES LIKE "glossary"');
+
+        $installed = !empty($tableExists);
+
+        if ($installed) {
```

Comment:
```suggestion
        if ($schema->hasTable('glossary')) {
```

Reply:
we also need a warning here that the bundle should be enabled manually in `config/bundles.php`.

Reply:
If the glossary table exists:
![image](https://user-images.githubusercontent.com/38959016/209081810-4d3b0eaa-371b-402b-bb71-d1f919a221b3.png)

Reply:
looks good.. can we also add the bundle name to the warning?

Reply:
Should we add the whole name ```Pimcore\Bundle\GlossaryBundle\GlossaryBundle```  or just ```GlossaryBundle```?

Reply:
We can do it like this
```php
  $this->warnIf(
          $schema->hasTable('glossary'),
          sprintf('Please make sure to enable the %s manually in config/bundles.php', GlossaryBundle::class)
  );
```